apps/base/utils.py

```diff
@@ -168,6 +168,29 @@ def get_boto3_client(resource, aws_keys):
         logger.exception(e)
 
 
+def get_presigned_url_for_file_upload(file_name, file_key):
+    """
+    Function to get the presigned url to upload a file to s3 with name file_name and key as file_key.
+    """
+    try:
+        aws_keys = {
+            "AWS_ACCESS_KEY_ID": settings.AWS_ACCESS_KEY_ID,
+            "AWS_SECRET_ACCESS_KEY": settings.AWS_SECRET_ACCESS_KEY,
+            "AWS_REGION": os.environ.get("AWS_DEFAULT_REGION", "us-east-1"),
```

Comment:
@KhalidRmb can you load the region in settings too. Doesn't make sense to do it using env variable here. Also please create a helper method `get_aws_secret_keys` use that to get this `dict`. The method can be used in future